lib/connection_pool.rb

```diff
@@ -111,7 +153,7 @@ def shutdown(&block)
   # removing it the pool. Subsequent checkouts will create new connections as
   # needed.
 
-  def reload(&block)
+  def reload(force: false, &block)
```

Comment:
Is this still necessary?

Reply:
Yes, to clear what might have been cached in the current thread storage.

Reply:
But you aren’t using it…

Reply:
Sorry, I somehow didn't get a notification for your comment.

Yes I use it in `after_fork`, here: https://github.com/mperham/connection_pool/pull/166/files#diff-af534256dda036ebb870c318564e254eedab4d10290508088f179cbc51971ff6R56

Reply:
But that's `checkin`, not `reload`.

Reply:
🤦 oh indeed. That's a leftover from a previous version where `after_fork` would call `reload`. I'll clean it up.

Reply:
Done.